aruco_detector_node:
  ros__parameters:
    image_topic: /camera/color/image_raw
    camera_info_topic: /camera/color/camera_info

    marker_length_m: 0.038
    id_left: 13
    id_right: 14

    x_thresh_cm: 5.0
    theta_thresh_deg: 5.0
    x_close_cm: 1.0
    theta_close_deg: 2.0
    target_theta_deg: 0.0
    camera_x_offset_cm: 4.0

    min_cmd_interval: 0.5

    target_z_cm: 49.0
    z_close_cm: 1.0
    approach_k: 0.6
    min_step_cm: 3.0
    max_step_cm: 12.0
    max_total_cm: 120.0

    insert_total_cm: 17.0
    insert_step_cm: 5.0
    lift_small_cm: 3.0
    retract_extra_cm: 0.0

    reacquire_miss_n: 3
    reacquire_hit_n: 2

    cmd_timeout_s: 6.0
    max_cmd_retries: 1
